### PR TITLE
start adding new attributes

### LINKED ISSUE #177: Add a new data attribute for the UI Renderer that accepts a bearer token, Fallback URL, Stage, License Plate, UUID, Policy Number, Partner number
bildschirmfoto 2018-06-06 um 10 32 24

```JavaScript

if (data.aletheiaOauthToken) {
  // reducer set oauth token and add header Authorization. ROID, CID and PROCESS ID wont be part of data
...
 xhttp.setRequestHeader('Authorization', data.aletheiaOauthToken);
...
} else {
 // use current flow. 
}

```

### PR DESCRIPTION
**Pull request by AndyOGo**
fixes #177

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -51,6 +51,14 @@
...
        data-aletheia
        data-aletheia-locale="de-CH"
+       data-aletheia-stage="<%= htmlWebpackPlugin.options.stage %>"
+       data-aletheia-app-id="<%= htmlWebpackPlugin.options.appId %>"
+       data-aletheia-partner-number="<%= htmlWebpackPlugin.options.partnerNumber %>"
+       data-aletheia-policy-number="<%= htmlWebpackPlugin.options.policyNumber %>"
+       data-aletheia-license-plate="<%= htmlWebpackPlugin.options.licensePlate %>"
+       data-aletheia-oauth-token="<%= htmlWebpackPlugin.options.oauthToken %>"
+       data-aletheia-uuid="<%= htmlWebpackPlugin.options.uuid %>"
+       data-aletheia-fallback-url="<%= htmlWebpackPlugin.options.fallbackUrl %>"
```
Comment (AndyOGo) [inline src/index.html]: @LucaMele 
@mjenny 
@lina-salih 
**Fallback-URL** is in discussion, but will rather be rejected by axa.ch team

File: src/index.html [hunk 2]
```diff
@@ -51,6 +51,14 @@
        class="js-aletheia-mount"
        data-aletheia
        data-aletheia-locale="de-CH"
+       data-aletheia-stage="<%= htmlWebpackPlugin.options.stage %>"
+       data-aletheia-app-id="<%= htmlWebpackPlugin.options.appId %>"
+       data-aletheia-partner-number="<%= htmlWebpackPlugin.options.partnerNumber %>"
+       data-aletheia-policy-number="<%= htmlWebpackPlugin.options.policyNumber %>"
+       data-aletheia-license-plate="<%= htmlWebpackPlugin.options.licensePlate %>"
```
Comment (AndyOGo) [inline src/index.html]: @mjenny 
@lina-salih 
@LucaMele 
**License-Plate** has never been and is not provided by myAXA!

File: src/index.html [hunk 3]
```diff
@@ -51,6 +51,14 @@
        class="js-aletheia-mount"
        data-aletheia
        data-aletheia-locale="de-CH"
+       data-aletheia-stage="<%= htmlWebpackPlugin.options.stage %>"
+       data-aletheia-app-id="<%= htmlWebpackPlugin.options.appId %>"
```
Comment (AndyOGo) [inline src/index.html]: @mjenny 
@LucaMele 
@lina-salih 
myAXA has another naming convention for the following data:
- `appl_id`
- `part_nr`
- `pol_nr`

File: src/redux/init/init.actions.js [hunk 4]
```diff
@@ -43,28 +53,44 @@ export const init = options => (dispatch, getState) => {
...
       },
       customData: {
         role: 'agent',
       },
       amr: '',
     }))
   const xAxaContext = `${jwt1}.${jwt2}.`
+
+  if (locale.indexOf('-') === -1) {
+    locale = `${locale}-CH`
```
Comment (LucaMele) [inline src/redux/init/init.actions.js]: attention as `en-CH` does not exist

**Pull request merged**